fixtures/eslint-v9/package.json

```diff
@@ -2,8 +2,9 @@
   "private": true,
   "name": "eslint-v9",
   "dependencies": {
-    "eslint": "^9",
-    "eslint-plugin-react-hooks": "link:../../build/oss-stable/eslint-plugin-react-hooks"
+    "eslint": "^9.18.0",
+    "eslint-plugin-react-hooks": "link:../../build/oss-stable/eslint-plugin-react-hooks",
+    "jiti": "^2.4.2"
```

Comment:
`jiti` is what `eslint` uses for parsing ts configs.  https://eslint.org/docs/latest/use/configure/configuration-files#typescript-configuration-files